grml-debootstrap

```diff
@@ -1692,6 +1692,11 @@ grub_install() {
     eend $?
   fi
 
+  if [ -n "${BOOT_APPEND}" ] ; then
+    echo "Adding BOOT_APPEND configuration ['${BOOT_APPEND}'] to /etc/default/grub."
+    sed -i "/GRUB_CMDLINE_LINUX_DEFAULT/ s#\"\$# ${BOOT_APPEND}\"#" "${MNTPOINT}/etc/default/grub"
+  fi
+
```

Comment:
Ah, aren't we executing `grub_install()` from chroot-script in the --vm mode?

Reply:
Apparently not. I was wondering about the code duplication in chroot-script<>grml-debootstrap.